cli/cmd/org/create.go

```diff
@@ -11,21 +12,40 @@ import (
 )
 
 func CreateCmd(cfg *config.Config) *cobra.Command {
-	var description string
+	var name, description string
 
 	createCmd := &cobra.Command{
-		Use:   "create <org-name>",
+		Use:   "create",
 		Short: "Create",
-		Args:  cobra.ExactArgs(1),
+		Args:  cobra.NoArgs,
 		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := cmd.Context()
+
 			client, err := cmdutil.Client(cfg)
 			if err != nil {
 				return err
 			}
 			defer client.Close()
 
+			if !cmd.Flags().Changed("name") {
+				// Get the new org name from user if not provided in the flag
+				err := cmdutil.PromptIfUnset(&name, "Org Name", "")
+				if err != nil {
+					return err
+				}
+			}
+
+			exist, err := cmdutil.OrgExists(ctx, client, name)
```

Comment:
We can avoid this RPC call and can directly call `CreateOrganization` which will fail if org name already exists.